### DISCUSSION TITLE
When using GFPGAN is it possible to just keep the final image in the samples folder instead of both?

### DISCUSSION THREAD

**Discussion by Wang-Thunder** [hlky/stable-diffusion#172] [Q&A] [answered]
Updated repo today and noticed that there is a copy of the pre/post GFPGAN image in the samples folder. Is there a setting somewhere to make it like the previous webui and just store the corrected image instead of both?

Thanks! :)

**Comment by tarasconus** [comment, marked as answer]
Just uncheck "Save individual images" under advanced, and it will only save the GFPGAN.

**Reply by Wang-Thunder** [reply]
~~It still outputs both files, even with it unchecked~~

Didn't notice the update. This issue was fixed apparently. If you have this problem, make sure you update ;)
